Client/Frontend/Browser/Tabs/InactiveTabViewModel.swift

```diff
@@ -92,7 +92,7 @@ class InactiveTabViewModel {
         let day14_Old = Calendar.current.date(byAdding: .day, value: -14, to: noon) ?? Date()
         let defaultOldDay = day14_Old
         
-//        let min_Old = Calendar.current.date(byAdding: .minute, value: -1, to: currentDate) ?? Date() // testing only
+//        let min_Old = Calendar.current.date(byAdding: .second, value: -10, to: currentDate) ?? Date() // testing only
 //        let defaultOldDay = min_Old
```

Comment:
Can we remove this commented code?

Reply:
I would prefer to keep it for easy testing purposes in the future so we don't have to keep writing it out. I could add a comment about that.

I dunno. WHat do you think?

Reply:
Ok with comment